packages/integrations/mdx/src/index.ts

```diff
@@ -46,7 +49,7 @@ export default function mdx(partialMdxOptions: Partial<MdxOptions> = {}): AstroI
 				addRenderer(astroJSXRenderer);
 				addPageExtension('.mdx');
 				addContentEntryType({
-					extensions: ['.mdx'],
+					extensions: extensions,
```

Comment:
don't know whether codebase prefers to use shorthand properties, erred on the side of explicitness here and didn't seem to be elided from linter/formatter in any case